src/osparc_python_runner/main.py

```diff
@@ -1,71 +1,32 @@
+import json
 import logging
 import os
-import pickle
 import shutil
 import subprocess
 import sys
-import tempfile
-import zipfile
 from pathlib import Path
+from typing import Dict
 
 logging.basicConfig(level=logging.INFO)
 logger = logging.getLogger("osparc-python-main")
 
 
 ENVIRONS = ["INPUT_FOLDER", "OUTPUT_FOLDER"]
 try:
-    input_dir, output_dir = [Path(os.environ[v]) for v in ENVIRONS]
+    INPUT_FOLDER, OUTPUT_FOLDER = [Path(os.environ[v]) for v in ENVIRONS]
 except KeyError:
     raise ValueError("Required env vars {ENVIRONS} were not set")
 
-# TODO: sync with schema in metadata!!
-OUTPUT_FILE = "output_data.zip"
+# NOTE: sync with schema in metadata!!
```

Comment:
MINOR: let's create an issue on this repo ... ? I might consider this service as an example when working in OOIL next week

Reply:
done